src/Datasource/Paginator.php

```diff
@@ -99,19 +99,19 @@ class Paginator implements PaginatorInterface
      * By default CakePHP will automatically allow sorting on any column on the
      * repository object being paginated. Often times you will want to allow
      * sorting on either associated columns or calculated fields. In these cases
-     * you will need to define a whitelist of all the columns you wish to allow
-     * sorting on. You can define the whitelist in the `$settings` parameter:
+     * you will need to define an allowed list of all the columns you wish to allow
+     * sorting on. You can define the allowed sort fields in the `$settings` parameter:
      *
      * ```
      * $settings = [
      *   'Articles' => [
      *     'finder' => 'custom',
-     *     'sortWhitelist' => ['title', 'author_id', 'comment_count'],
+     *     'allowedSort' => ['title', 'author_id', 'comment_count'],
```

Comment:
```suggestion
     *     'allowedSortFields' => ['title', 'author_id', 'comment_count'],
```
If we're going to make it clearer, should we specify that they are fields or are they allowed to be expressions?

Reply:
This list has to be strings as it is for checking the URL parameters for safeness.

Reply:
```suggestion
     *     'sortableFields' => ['title', 'author_id', 'comment_count'],
```

`allowedSort` doesn't sound naturally

Reply:
I too like `sortableFields`.

Reply:
I'll get this updated to sortableFields :+1:

Reply:
Done now.